tensorflow/compiler/jit/mark_for_compilation_pass_test.cc

```diff
@@ -1379,7 +1371,7 @@ TEST(XlaCompilationTest, CreateCombinedCpuGpuClusters) {
 
   std::unordered_map<string, string> clusters = GetClusters(*graph);
 
-  EXPECT_NE(clusters["test/x"], "");
+  EXPECT_EQ(clusters["test/x"], "");
```

Comment:
You can delete this test case, now that it's not supported (this is just checking that nothing was clustered).